1/styles/scss/_media-mobile.scss

```diff
@@ -0,0 +1,158 @@
+@media screen and (max-width: 576px) {
+  .container{
+    max-width: 80vw;
+  }
+  .heading h1{
+    font-size: 40px;
+    padding: 140px 0;
+  }
+  .header-menu .active::after{
+    left: 45%;
+  }
+  .logo{
+    padding: 20px 30px;
+  }
+  .local-nav {
+    z-index: 0;
+    left: 13%;
+    li {
+      padding: 10px 25px;
+    }
+  }
+  .tips{
+    .header{
+      margin-top: 30px;
+      svg{
+        width: 160px;
+      }
+    }
+  }
+  .accordion{
+    margin-top: 20px;
+
+    .number{
+      padding: 0 10px;
+      position: relative;
+      top: 10px;
+    }
+    .accordion-item-header{
+      padding: 0 0 20px 0;
+      font-size: 16px;
+      align-items: flex-end;
+    }
+    .plus{
```

Comment:
![image](https://user-images.githubusercontent.com/58749210/119579166-8788dd00-bdc6-11eb-8915-d5b8608d0607.png)
doesn't match

Reply:
done

Reply:
red areas don't match the design 
![image](https://user-images.githubusercontent.com/84564899/119956692-08ec9500-bfaa-11eb-8c18-1a604ce627d0.png)

Reply:
done